src/Resources/contao/drivers/DC_Folder.php

```diff
@@ -1126,6 +1133,11 @@ public function move($blnIsAjax=false)
 			// Redirect or reload
 			if (!$objUploader->hasError())
 			{
+				if ($blnIsAjax)
+				{
+					throw new ResponseException(new Response(\Message::generateUnwrapped(), 201));
```

Comment:
I am only going to merge this if all @contao/developers confirm that this is correct in their opinion, because I definitely think that you are wrong.